Added document section with required Links (#143)
anitab-org/anitab-forms-web: README.md

Documentation for the project is hosted [here](https://osp-web-docs.surge.sh/). We use `Docusaurus` for maintaining the documentation of the project.

### Few Useful Links:

> 1. **`anitab-org wiki`** link - https://github.com/anitab-org/anitab-forms-web/wiki

Reviewer: ```suggestion
1. **anitab-org wiki** [link](https://github.com/anitab-org/anitab-forms-web/wiki)
```
Do the same with other links also

Reviewer 2: Ok

> 1. [anitab-org wiki](https://github.com/anitab-org/anitab-forms-web/wiki) : Wiki page  

Reviewer 3: ```suggestion
1. [anitab-forms wiki](https://github.com/anitab-org/anitab-forms-web/wiki) : Wiki page  
```

> 1. [anitab-forms wiki](https://github.com/anitab-org/anitab-forms-web/wiki) : Wiki page  
> 2. [Open Sessions](https://meet.google.com/eqb-nuut-kqm) : You can join Meetups organized by anitab-org through this link.
> 3. [GSoC youtube video link](https://youtu.be/3A746GppZ0Y) : GSoC 2020 final demo - anitab-forms youtube video link. 
> 4. [Setting up the project youtube video link](https://youtu.be/_b2RQGbYN9w) : Here is the youtube video link for setting up the project for development purpose.
> 5. [Question in Zulip chats link](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions) : In this zulip channel , all the Q&A happens related to anitab-org opensource.
> 6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements) : In this stream, all the important decision are announced by admins related to anitab-org opensource.
> 

Reviewer 4: ```suggestion
1. [anitab-forms wiki](https://github.com/anitab-org/anitab-forms-web/wiki): Wiki page  
2. [Open Sessions](https://meet.google.com/eqb-nuut-kqm): Link to join weekly meetings for anitab-forms.
3. [GSoC youtube video link](https://youtu.be/3A746GppZ0Y): Final video demo of anitab-forms developed during GSoC 2020
4. [Setting up the project youtube video link](https://youtu.be/_b2RQGbYN9w): Here is the youtube video link for setting up the project for development purposes.
5. [Question in Zulip chats link](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions): In this zulip channel, all the Q&A happens related to anitab-org opensource.
6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements): In this stream, all the important decisions are announced by admins related to anitab-org opensource.

```

Documentation for the project is hosted [here](https://osp-web-docs.surge.sh/). We use `Docusaurus` for maintaining the documentation of the project.

### Few Useful Links:
1. **anitab-org wiki** : [Visit](https://github.com/anitab-org/anitab-forms-web/wiki)

> 2. **Open Sessions** : [Visit](https://meet.google.com/eqb-nuut-kqm)

Reviewer 4: @Aaishpra Is an open session link necessary in this? Because this may change in future.

Reviewer: Yes, it can definitely change, Shall we add the document link of our open sessions instead of this?

3. **GSoC** youtube video link : [Visit](https://youtu.be/3A746GppZ0Y)
4. **Setting up the project** youtube video link : [Visit](https://youtu.be/_b2RQGbYN9w)
5. **Question in Zulip chats** link : [Visit](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions)

> 6. **Announcement in Zulip chats** link : [Visit](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements)

Reviewer 4: ```suggestion
6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements): In this stream, all the important decision are announced by admins related to anitab-org opensource.
```
What do you think @Aaishpra about this representation i.e. link followed by some detail? This can be done for all the links.

Reviewer 2: Can I do this changes?

Reviewer: This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

Reviewer: Although we can definitely add details for other streams.
@StarTrooper08 can you add these details by yourself or would you like some help?

Reviewer 2: > This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

I have one idea ,
we can just add one zulip chat link instead of those 2 channels links of zulip chat.

What do you say??

Reviewer: Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
Let's see what @codesankalp says

Reviewer 2: > Although we can definitely add details for other streams.
> @StarTrooper08 can you add these details by yourself or would you like some help? 

Ok I will first try of my own and if stuck somewhere I will take help .

Reviewer 2: > Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
> Let's see what @codesankalp says

Ok
